### PR TITLE
native index statements

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/14.native-index-statements/2.1.show-create-index.md [hunk 1]
```diff
@@ -32,22 +30,25 @@ nebula> SHOW CREATE TAG INDEX player_index_1;
...
-| "index_follow"  | "CREATE EDGE INDEX `index_follow` ON `follow` ( |
-|                 |  `degree`                                       |
+| "follow_index"  | "CREATE EDGE INDEX `follow_index` ON `follow` ( |
 |                 | )"                                              |
 +-----------------+-------------------------------------------------+
 ```
+
+!!! Compatibility "Legacy version compatibility"
+
+    In Nebula Graph 2.0.1, the `SHOW TAG/EDGE INDEXES` statement only returns `Names`.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/14.native-index-statements/2.1.show-create-index.md:54]: 2.0.1
is it macro? or hardcode

File: docs-2.0/3.ngql-guide/14.native-index-statements/2.show-native-indexes.md [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
++----------------+----------+---------+
+| Index Name     | By Edge  | Columns |
++----------------+----------+---------+
+| "follow_index" | "follow" | []      |
++----------------+----------+---------+
 ```
+
+!!! Compatibility "Legacy version compatibility"
+
+    In Nebula Graph 2.0.1, the `SHOW TAG/EDGE INDEXES` statement only returns `Names`.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/14.native-index-statements/2.show-native-indexes.md:37]: ditto

File: docs-2.0/3.ngql-guide/14.native-index-statements/2.1.show-create-index.md (hunk 1 above)
Comment (izhuxiaoqing) [reply docs-2.0/3.ngql-guide/14.native-index-statements/2.1.show-create-index.md:54]: Hardcode. And in 2.0.1, this statement only returns names.

File: docs-2.0/3.ngql-guide/14.native-index-statements/2.show-native-indexes.md (hunk 2 above)
Comment (izhuxiaoqing) [reply docs-2.0/3.ngql-guide/14.native-index-statements/2.show-native-indexes.md:37]: Same as above

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**